mycroft/tts/mozilla_tts.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import hashlib
+import os
+import requests
+
+from .tts import TTS, TTSValidator
+from mycroft.configuration import Configuration
+from mycroft.util.log import LOG
```

Comment:
Seems like now os, hashlib and mycroft.util.log isn't used. Apart from those tiny things it looks great!